### PR TITLE
Add fake server test for Negotiate authentication.

### PR DESCRIPTION
**Pull request by filipnavara**
Fix incorrect computation of IsCompleted in managed NTLM implementation.

### CODE REVIEW

File: src/libraries/Common/src/System/Net/NTAuthentication.Managed.cs [hunk 1]
```diff
@@ -316,7 +316,7 @@ internal void CloseContext()
             {
                 Debug.Assert(decodedIncomingBlob != null);
 
-                if (_isSpNego)
+                if (!_isSpNego)
```
Comment (filipnavara) [inline src/libraries/Common/src/System/Net/NTAuthentication.Managed.cs:319]: This condition was incorrectly flipped.

**Comment by filipnavara** [comment]
Keeping as draft since the System.Net.Security tests currently don't run with the managed implementation on either Android or tvOS.

**Review by wfurt — APPROVED** [review]
LGTM

**Comment by filipnavara** [comment]
@wfurt I'll try to wire up the mobile tests in a follow up PR(s), so this should be good to go as-is.

**Pull request merged**